manifest.yml.sample

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Would it be possible to put a comment here explaining that this is for CloudFoundry?  Or maybe have this in deployment/?